framework/PostProcessors/BasicStatistics.py

```diff
@@ -270,7 +241,12 @@ def _handleInput(self, paramInput):
     for child in paramInput.subparts:
       tag = child.getName()
       #because percentile is strange (has an attached parameter), we address it first
+      if tag in ['percentile'] + self.scalarVals + self.vectorVals:
+        if 'prefix' not in child.parameterValues:
+          self.raiseAnError(IOError, "Not prefix is provided for node: ", tag)
+        prefix = child.parameterValues['prefix']
       if tag == 'percentile':
+        #get the prefix
```

Comment:
this comment should be probably moved up

Reply:
fixed.